code/_core/obj/item/soulgems/_soulgem.dm

```diff
@@ -156,4 +179,11 @@
 
 /obj/item/soulgem/godly/filled/Generate()
 	. = ..()
-	total_charge = total_capacity
\ No newline at end of file
+	total_charge = total_capacity
+
+/obj/item/soulgem/azuras_star
+	total_capacity = SOUL_SIZE_MYSTIC
+	is_star = TRUE
+/obj/item/soulgem/azuras_star/filled/Generate()
+	. = ..()
+	total_charge = total_capacity
```

Comment:
Should probably remove this. Has no reason to spawn full, honestly.

Reply:
Good point